_episodes/04-tabular_data.md

```diff
@@ -332,26 +334,30 @@ Wait!  That's not right!  What is wrong with that output?  How can we fix it?
 >>
 >> ~~~
 >> import numpy
+>>
 >> xyz_file = numpy.genfromtxt(fname='water.xyz', skip_header=2, dtype='unicode')
 >> symbols = xyz_file[:,0]
 >> coord = (xyz_file[:,1:])
 >> coord = coord.astype(numpy.float)
+>>
 >> for numA, atomA in enumerate(coord):
 >>     for numB, atomB in enumerate(coord):
->>        BL_AB = numpy.sqrt((atomA[0]-atomB[0])**2+(atomA[1]- atomB[1])**2+(atomA[2]-atomB[2])**2)
->>        BL_AB = round(BL_AB,3)
->>        print(symbols[numA], "to", symbols[numB], ":", BL_AB)
+>>        bond_length_AB = numpy.sqrt((atomA[0]-atomB[0])**2+(atomA[1]- atomB[1])**2+(atomA[2]-atomB[2])**2)
+>>        print("%s to %s : %.3f" %(symbols[numA], symbols[numB], bond_length_AB)))
```

Comment:
I think this printing syntax is more confusing, plus it hasn't been introduced before.